pkg/types/clusterquota.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+import (
+	"github.com/openshift/installer/pkg/types/gcp"
+)
+
+// ClusterQuota contains the size, in cloud quota, of
+// the cluster that was created by installer.
+type ClusterQuota struct {
+	GCP *gcp.Quota `json:"gcp,omitempty"`
```

Comment:
why are these new. these shouldn't have shown up if you had rebased the branch.

Reply:
hm. will look into. thanks